hashicorp-vault/renew-ssh-host-certificate.sh

```diff
@@ -2,33 +2,64 @@
 
 set -e
 
-# Get the role ID
-roleid=$(cat /opt/secrets/roleid.txt)
+# Check if jq is installed
+if ! command -v jq &> /dev/null; then
+    echo "Error: 'jq' is not installed. Please install it to use this script."
+    exit 1
+fi
+
 # Get the SSH public key
 ssh_public_key=$(cat /etc/ssh/ssh_host_ed25519_key.pub)
+config_file=config.json
+
+# Check if the configuration file exists
+if [[ ! -f "$config_file" ]]; then
+    echo "Error: Configuration file '$config_file' not found!"
+    exit 2
+fi
+
+server=$( jq -r .server "$config_file" )
+roleid=$( jq -r .roleid "$config_file" )
+port=$( jq -r .port "$config_file" )
+sshEnginePath=$( jq -r .sshEnginePath "$config_file" )
+hostRole=$( jq -r .hostRole "$config_file" )
+appRole=$( jq -r .appRole "$config_file" )
```

Comment:
I like the idea with a config file. But considering this block, it might be easier to use an environment file and just use `source config.env`?